db/migrate/20210127021909_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+      t.string :password
```

Comment:
I forgot to mention to you that all these columns should be required in the database. Please update this same migration but make sure you drop the current database schema.